articles/logic-apps/quickstart-create-first-logic-app-workflow.md

```diff
@@ -112,11 +112,11 @@ Now add an [action](../logic-apps/logic-apps-overview.md#logic-app-concepts) tha
    * For Azure work or school accounts, select Office 365 Outlook.
    * For personal Microsoft accounts, select Outlook.com.
 
-1. If asked for credentials, sign in to your email account so that Logic Apps creates a connection to your email account.
+1. If asked for credentials, sign in to your email account so that logic apps creates a connection to your email account.
```

Comment:
Actually, this instance refers to the service name, which is capitalized, so please change to "Azure Logic Apps". Thanks!